### ISSUE TITLE
How to manually set the Superview

### ISSUE THREAD

**Issue by MutantPigeon24** [IjzerenHein/autolayout.js#8]
I understand that the pipe character '|' is used to represent the superview. Is there a way to specify the superview in the EVFL.

I've tried many times using the live editor [live editor](https://rawgit.com/IjzerenHein/visualformat-editor/master/dist/index.html?vfl=example) but so far, no luck.

Some silly pseudo code to get my point across :)
````
HV:|~[parent(100)~|
[child]:superview=parent 
HV:|-(50)-[child]
`````

In the above scenario, **child** would be 50px away from **parent's** 0,0.
Visually it'd have an X and Y that is 50px larger than **parent**, but in actual fact, it's X and Y would be 50, because it is a child of **parent**.

tl;dr How do I set the superview of a element.

**Comment by IjzerenHein** [comment]
Well, simply put, you can't. You can't set the parent of a sub-view. All sub-views are direct children of the one super-view. If you want a hierarchy then you could create multiple VFL definitions and create a view for each and layout a view from a view.

Alternatively, you can create so-called "view-stacks" and group vertical or horizontal elements together:
https://github.com/IjzerenHein/autolayout.js#view-stacks
